### ISSUE TITLE
I'm having trouble setting up the development environment 

### ISSUE THREAD

**Issue by jpbelang** [dkandalov/limited-wip#24]
How do you setup the environment ?  I'm having trouble getting it to work (if goes back to my home directory).

Gradle seems to work, but the project seems to interfere with gradle.

**Comment by dkandalov** [comment]
Oh, sorry. The master branch is work in progress right now because of
IntelliJ API changes.

I don’t remember having problems setting it up thought. Which version are
you using?

**Comment by jpbelang** [comment]
2019.1 (Ultimate) I use the built-in repo startup, which I think falls onto your .idea directory.  I then load the gradle files.  The gradle build works.  But when I reload the project, it's all screwed up.

**Comment by dkandalov** [comment]
Sorry, I just got around to try it. Thanks a lot for reporting the issue!

I think I managed to reproduce something that roughly matches your description (in #IU-191.7141.44):
 - git clone, open in IJ, choose all the defaults in gradle import dialog
 - attempt to compile, it fails so I have to choose JDK in projects settings
 - attempt to compile from IDE which fails complaining it cannot find classes like `kotlin.Int` even though they are not red in IDE.

What seems to work is deleting `.idea` and `limitedwip.iml` files and doing gradle import again.

I suspect the reason for this is that github has old IDE files and IJ fails to read them properly ending up in some broken state. I'll remove them or will commit new files (the main motivation for committing them is to disable some inspections so that they don't complain about code style).
